### PR TITLE
More var to let and const

### PR DESCRIPTION
**Pull request by willeastcott**
More removal of the `var` keyword, largely focused on the framework sourcebase.

I confirm I have read the [contributing guidelines](https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md) and signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: src/framework/application.js [hunk 1]
```diff
@@ -1748,11 +1739,11 @@ class Application extends EventHandler {
      * });
      */
     renderLine(start, end, color) {
-        var endColor = color;
-        var options;
+        let endColor = color;
```
Comment (mvaligursky) [inline src/framework/application.js:1742]: these render functions are all now moved to deprecated in my other PR.

Comment (willeastcott) [reply src/framework/application.js:1742]: So what needs to be done here?

Comment (mvaligursky) [reply src/framework/application.js:1742]: nothing for now I think .. we'll resolve to conflict when we get to it.

Comment (willeastcott) [reply src/framework/application.js:1742]: ok

**Review by mvaligursky — APPROVED** [review]

**Pull request merged**